tests/integration/event_source_kafka/test_kafka_rules.yml

```diff
@@ -1,7 +1,7 @@
 - name: test kafka source plugin
   hosts: localhost
   sources:
-    - ansible.eda.kafka:
+    - kafka:
```

Comment:
I'm not sure why you have removed the FQCN here, but "sources" option, even if it is tracked in the CLIRunner is only for development purposes and should not be used in these tests, can you come back to "ansible.eda.kafka" ?

Reply:
When I run tests locally with the FQCN (`ansible.eda.kafka`) and the default sources directory (`None`), they all fail with the following error:

```
2023-10-27 10:16:20,963 - ansible_rulebook.engine - ERROR - Source error Could not find source plugin for ansible.ed...ble.eda.kafka
2023-10-27 10:16:20,980 - ansible_rulebook.cli - ERROR - Terminating One of the source plugins failed
```

When I add the `-S extensions/eda/plugins/event_source` argument, they fail too with the same error.

Tests succeed only if the Kafka source plugin is renamed to `kafka` (without FQCN) and with the overridden sources directory.

Do you have a solution?

Reply:
Hi @jouir certainly is weird, It should work. Have you check in the collection is properly installed before running ansible-rulebook? The pipeline should not fail. 
https://github.com/ansible/event-driven-ansible/blob/main/.github/workflows/integration-tests.yml#L43

Reply:
Adding the collection locally using ansible-galaxy was the trick. Thanks!